roles/branch-validation/tasks/main.yml

```diff
@@ -155,7 +167,7 @@
     that: ping.user_input|bool
 
 - include: ha.yml
-  when: validate_ha_environment
+  when: not groups['compute'] | intersect(groups['controller']) | length | bool
```

Comment:
Can you help me understand what this conditional is trying to accomplish?

We want to validate ha, regardless if the environment is hyper converged (compute on controllers) or not.

What we need to do is adjust the tasks inside ha.yml so that there aren't any assumptions made about where the booted instances are.